package.json

```diff
@@ -8,12 +8,18 @@
     "url": "https://github.com/apipatth/ppoker-rest-react.git"
   },
   "dependencies": {
+    "@types/jest": "^24.0.23",
```

Comment:
should we add these as `devDependencies` instead?

Reply:
It's as per CRU. I can change to devDependency

Reply:
aaah I see, maybe it's to be able to build the app even when running `yarn install` with `NODE_ENV=production`, let's keep them as dependencies :)